packages/core/src/Container/Container.tsx

```diff
@@ -41,4 +41,4 @@ Container.propTypes = propTypes
 
 Container.displayName = 'Container'
 
-export default Container
+export { Container }
```

Comment:
does the index file have to be updated for this?

Reply:
it was. Vite's bundler is very specific about ambiguity between imports. all the changes that look like this are for that reason.